docs/_scripts/download_after_sync.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+DOCKER_TAG='latest'
+
+echo "Setting REGISTRY_PATH, which can be used directly with the Docker Client."
+export REGISTRY_PATH=$(http $BASE_ADDR$DISTRIBUTION_HREF | jq -r '.registry_path')
+
+# If Pulp was installed without CONTENT_HOST set, it's just the path.
+# And httpie will default to localhost:80
+if [[ "${DISTRIBUTION_BASE_URL:0:1}" = "/" ]]; then
```

Comment:
i cannot find where it is defined?

Reply:
This was a copy paste error, removed now.